tst/test_schedulers.py

```diff
@@ -134,6 +143,14 @@ def test_sync_scheduler(searcher, mode):
         max_resource_attr="steps",
         random_seed=random_seed,
     )
+    if scheduler == "hb":
+        myscheduler = SynchronousGeometricHyperbandScheduler(
+            config_space, **scheduler_kwargs
+        )
+    else:
+        myscheduler = GeometricDifferentialEvolutionHyperbandScheduler(
+            config_space, **scheduler_kwargs
+        )
```

Comment:
nit: you could add the class names directly into parametrize and then you would not need to have this distinction here

Reply:
ok